unison-src/new-runtime-transcripts/hashing.md

```diff
@@ -15,7 +15,7 @@ You can skip this section, which is just needed to make the transcript self-cont
 .builtin> ls Bytes
 ```
 
-Notice the `fromBase16` and `toBase16` functions. Here's some (somewhat inefficient) convenience functions for converting `Bytes` to and from base-16 `Text`. These could be replaced by use of `Text.toUtf8` and `Text.tryFromUtf8` once those builtins exist:
+Notice the `fromBase16` and `toBase16` functions. Here's some (somewhat inefficient) convenience functions for converting `Bytes` to and from base-16 `Text`. These could be replaced by use of `Text.toUtf8` and `Text.tryFromUtf8`.
```

Comment:
Can you simplify this transcript now? toBase16 can just be passed to toUtf8 for encoding to text, and for decoding you just utf8 decode and then pass that to fromBase16. 

You can maybe get rid of the List.map definition if it’s not used elsewhere.